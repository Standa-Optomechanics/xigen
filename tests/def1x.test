#!/bin/sh
file_input="$srcdir/full.xi"
. $srcdir/defs

cat < $file_expected > $file_ok

expect_out_def
