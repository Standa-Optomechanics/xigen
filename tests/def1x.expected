	command_move @1
	command_movr @2
	set_encoder @3
	command_left @4
	command_right @5
	command_stop @6
	command_zero @7
	command_home @8
	command_loft @9
	get_additional_status @10
	set_dc_information @11
	get_dc_information @12
	set_dc_settings @13
	get_dc_settings @14
	set_step_information @15
	get_step_information @16
	set_step_settings @17
	get_step_settings @18
	set_encoder_information @19
	get_encoder_information @20
	set_encoder_settings @21
	get_encoder_settings @22
	set_gear_information @23
	get_gear_information @24
	set_gear_settings @25
	get_gear_settings @26
	set_engine_settings @27
	get_engine_settings @28
	set_secure_settings @29
	get_secure_settings @30
	set_edges_settings @31
	get_edges_settings @32
	set_pid_settings @33
	get_pid_settings @34
	set_sync_settings @35
	get_sync_settings @36
	set_extio_settings @37
	get_extio_settings @38
	set_home_settings @39
	get_home_settings @40
	command_read_settings @41
	command_save_settings @42
	command_eeread_settings @43
	command_eesave_settings @44
	get_read_uart @45
	set_write_uart @46
	get_serial_number @47
	get_bootloader_version @48
	get_firmware_version @49
	get_analog_data @50
	get_chart_data @51
	set_calibration_settings @52
	get_calibration_settings @53
	set_serial_number @54
	command_calibrate @55
